test/units/testsuite-46.sh

```diff
@@ -512,6 +512,8 @@ ExecStart=-/usr/sbin/sshd -i -d -e
 StandardInput=socket
 StandardOutput=socket
 StandardError=journal
+RuntimeDirectory=sshd
+RuntimeDirectoryMode=0755
```

Comment:
hm, the test already has `mkdir -p /usr/share/empty.sshd /var/empty /var/empty/sshd` to create this empty dir for various distros. i'd just add /run/sshd there too